packages/beacon-node/src/network/processor/gossipHandlers.ts

```diff
@@ -250,10 +256,20 @@ function getDefaultHandlers(modules: ValidatorFnsModules, options: GossipHandler
         // Returns the delay between the start of `block.slot` and `current time`
         const delaySec = chain.clock.secFromSlot(signedBlock.message.slot);
         metrics?.gossipBlock.elapsedTimeTillProcessed.observe(delaySec);
+        getBlockInput.prune();
       })
       .catch((e) => {
         if (e instanceof BlockError) {
           switch (e.type.code) {
+            case BlockErrorCode.DATA_UNAVAILABLE: {
+              // TODO: create a newevent unknownBlobs and only pull blobs
```

Comment:
yes, please create a new event here, it looks confused when we have the `signedBlock` here and still issue `unknownBlock` event 😅

Reply:
would like to do it in followup PR, will add in the deneb tracker to keep track

Reply:
updated in the tracker  (https://github.com/ChainSafe/lodestar/issues/5279) to tackle on top of the blob inclusion proof PR